dashboard/lib/policies/lti.rb

```diff
@@ -76,6 +79,14 @@ def self.get_account_type(roles)
     return User::TYPE_STUDENT
   end
 
+  # Returns true if any of the user's roles is the LTI instructor role
+  def self.lti_teacher?(roles)
+    Array(roles).each do |role|
+      return true if TEACHER_ONLY_ROLES.include? role
+    end
+    return false
```

Comment:
There's a shorthand for intersections in Ruby that might make this function slightly more concise
```suggestion
    (Set.new(roles) & TEACHER_ONLY_ROLES).any?
```